### ISSUE TITLE
[BUG] Not able to set header status value

### ISSUE THREAD

**Issue by thathi** [Microsoft/EasyRepro#755]
### [Not covered](https://github.com/microsoft/EasyRepro#crm-functionality-not-covered) functionality, feature requests, and questions should use the Feature Request or Question templates.

**EasyRepro Version**
  - [ ] Microsoft Dynamics 365 Online Version 8.2 (8.2.x) (DB 8.2.x) online
  - [ ] Microsoft Dynamics 365 Online Version 9.0 (9.0.x) (DB 9.0.x) online
  - [ ] Microsoft Dynamics 365 Online Version 9.0.2 (9.0.2) (DB 9.0.2) online
  - [x] Microsoft Dynamics 365 Online Version 9.1 (9.1.x) (DB 9.1.x) online
  
**UCI or Classic Web**
  - [x] UCI (XrmApp)
  - [ ] Classic Web (XrmBrowser)

**Online or On Premise**
  - [x] Online
  - [ ] On Premise

**Browser**
  - [x] Chrome
  - [ ] Firefox
  - [ ] IE
  - [ ] Edge

**Describe the bug**  
Not able to change the status of the Lead

**Special formatting / display**  

**Code to reproduce**  

                xrmApp.OnlineLogin.Login(_xrmUri, _username, _password);
                xrmApp.Navigation.OpenApp(UCIAppName.Sales);
                xrmApp.Navigation.OpenSubArea("Sales", "Leads");
                xrmApp.CommandBar.ClickCommand("New");
                xrmApp.ThinkTime(5000);
                xrmApp.Entity.SetValue("subject", TestSettings.GetRandomString(5, 15));
                xrmApp.Entity.SetValue("firstname", TestSettings.GetRandomString(5, 10));
                xrmApp.Entity.SetValue("lastname", TestSettings.GetRandomString(5, 10));
                xrmApp.Entity.Save();
                xrmApp.Entity.SetHeaderValue(new** OptionSet { Name = "statuscode", Value = "Contacted" });
                xrmApp.Entity.Save();
          
 
**Expected behavior**  
Status should be able to changed to "Contacted"

**Screenshots**  
![image](https://user-images.githubusercontent.com/15817637/74084648-f2df5c00-4a68-11ea-94b8-397f2dfab7e1.png)

**Additional context**  
Updated the following code and it is working as expected. 
Old Code: 
{ "Entity_Header_OptionSetFieldContainer", "//div[@data-id='header_[NAME].fieldControl-option-set-container']" },
Update Code:
{ "Entity_Header_OptionSetFieldContainer", "**//*[contains(@id, '[NAME]-FieldSectionItemContainer')]**" },

**Comment by thathi** [comment]
@AngelRodriguez8008 can you have a look at this please?

**Comment by AngelRodriguez8008** [comment]
Hi  @thathi,

You find find the right solution, but i prefer to change the path to Top container like this:
```csharp
{ "Entity_Header_OptionSetFieldContainer", "//div[@data-id='header_[NAME]']" }
```
I fixed it in PR #754. Can you test it?

Thanks & br

**Comment by AngelRodriguez8008** [comment]
Hi @thathi,

The PR is merged can we close this Issue?

Thanks & br
